src/Components/logic/Operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from "big.js";
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+     try {
+      return one.mod(two).toString();
+    } catch (err) {
+      return "Can't find modulo as can't divide by 0.";
+    }
+  }
```

Comment:
The division and modulo functions are not working properly.
- [ ] The division function says to throw an error "Can't divide by 0" when such an operation is carried out , but instead , it says "Infinity".
- [ ] The modulo function doesn't work at all, please fix this.